doc/admin-guide/files/records.yaml.en.rst

```diff
@@ -4454,12 +4454,11 @@ HTTP/2 Configuration
    Dynamic Table, however, headers still can be encoded as indexable
    representations. The upper limit is 65536.
 
-.. ts:cv:: CONFIG proxy.config.http2.max_header_list_size INT 131072
+.. ts:cv:: CONFIG proxy.config.http2.max_header_list_size INT 32768
    :reloadable:
 
    This advisory setting informs a peer of the maximum size of header list
-   that the sender is prepared to accept blocks. The default value, which is
-   the unsigned int maximum value in |TS|, implies unlimited size.
```

Comment:
Was this default statement simply incorrect?

Reply:
Yes, the sentence was not updated on #5319.